api/tests/unit/core/test_helpers.py

```diff
@@ -1,27 +1,75 @@
+from typing import TYPE_CHECKING
+
+import pytest
 from core.helpers import get_current_site_url
 from django.contrib.sites.models import Site
 
+if TYPE_CHECKING:
+    from pytest_django.fixtures import SettingsWrapper
+else:
+    SettingsWrapper = None
```

Comment:
Interesting. I've not seen this before, historically, I've done: 

```
if TYPE_CHECKING:
    from pytest_django.fixtures import SettingsWrapper

def test_something(settings: "SettingsWrapper"):
    ...
```

Is there a benefit to having this `else` clause here instead?

Reply:
Both are valid, I just prefer less quotes.